packages/expect/src/spy_matchers.js

```diff
@@ -54,6 +54,32 @@ const createToBeCalledMatcher = matcherName => (received, expected) => {
   return {message, pass};
 };
 
+const createToReturnMatcher = matcherName => (received, expected) => {
+  ensureNoExpected(expected, matcherName);
+  ensureMock(received, matcherName);
+
+  const receivedIsSpy = isSpy(received);
+  const type = receivedIsSpy ? 'spy' : 'mock function';
+  const receivedName = receivedIsSpy ? 'spy' : received.getMockName();
+  const count = receivedIsSpy
+    ? received.returnValues.count()
+    : received.mock.returnValues.length;
```

Comment:
@cpojer I blamed the spy switching we're doing here to you - is this only to support jasmine spies or am I missing something? If it's just jasmine, I think we can remove this switch in the new returnValues matchers since jasmine does not store those values

Reply:
Yes, this is just for Jasmine.